src/sbt-test/sbt-scalafix/explicit-files/.scalafix.conf

```diff
@@ -1,2 +1,2 @@
-rules = [DisableSyntax]
+rules = [DisableSyntax, RemoveUnused]
```

Comment:
this is needed to trigger the "semanticdb not found" error from the original bug report